Airwallex/Payment/Sources/PaymentMethodProvider.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  PaymentMethodProvider.swift
+//  Airwallex
+//
+//  Created by Weiping Li on 2024/12/12.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+@MainActor
+final class PaymentMethodProvider {
+    private let provider: AWXPaymentMethodListViewModel
+    enum Foo: String {
+        case one
+    }
+    var session: AWXSession {
+        provider.session
+    }
+    
+    private(set) var methods = [AWXPaymentMethodType]()
+    private(set) var consents = [AWXPaymentConsent]()
+    
+    init(provider: AWXPaymentMethodListViewModel) {
+        self.provider = provider
+    }
+    
+    var isApplePayAvailable: Bool {
+        guard let applePay = methods.first(where: { $0.name == AWXApplePayKey }) else {
```

Comment:
Can just be `methods.contains(where: { $0.name == AWXApplePayKey })`?

Reply:
done